app/org/sagebionetworks/bridge/dynamodb/DynamoStudyDao.java

```diff
@@ -59,14 +60,39 @@ public Study getStudy(String identifier) {
         if (study == null) {
             throw new EntityNotFoundException(Study.class, "Study '"+identifier+"' not found.");
         }
+        if (!study.isActive()) {
+            throw new EntityNotFoundException(Study.class, "Study '"+identifier+"' is de-activated.");
+        }
```

Comment:
From the outside world, the error message should be same: "Study 'foo' not found." For all intents and purposes, it has been "deleted" for everyone but admins. You can combine these two tests.